src/codeql.ts

```diff
@@ -21,29 +21,36 @@ export {
 // This name must match that used by the vscode extension when creating the pack.
 const REMOTE_QUERY_PACK_NAME = "codeql-remote/query";
 
+interface RunQueryResult {
```

Comment:
We should probably export this interface considering it's part of the public interface of this module.